packages/client/src/components/docs/functions-doc.tsx

```diff
@@ -0,0 +1,351 @@
+import { Fragment, useState } from "react";
+
+import { Accordion } from "../accordion";
+import { Example } from "../doc-example";
+import { Code } from "../doc-inline-code";
+import { Message } from "../doc-message";
+import { IDocPageProps } from "./types";
+
+export const FunctionsDoc = (props: IDocPageProps) => {
+    const [current, setCurrent] = useState("");
+
+    return (
+        <Fragment>
+            <h1 className="doc-title">Python Functions:</h1>
+
+            <h2 className="doc-subtitle">General Functions</h2>
+            <Accordion
+                title="print()"
+                sectionId="print()"
+                pageId={props.pageId}
+                click={(next: string) => {
+                    props.onSectionChange(current, next);
+                    setCurrent(next);
+                }}
+                current={current}
+            >
+                <p>Outputs the textual value to the console</p>
+                <Message>
+                    <p>
+                        Parameters: <Code>print(x)</Code>
+                    </p>
+                </Message>
+                <Example
+                    code={
+                        `print("Hello World") #outputs Hello World to the console`
+                    }
+                ></Example>
+                <p>To print multiple variables inside the text. You can use <Code>f''</Code> operator (Formattable Text)</p>
+                <p>Inside of the <Code>f''</Code> operator, to convert variables to its textual value, use the <Code>{}</Code> operator</p>

```

Comment:
I would first use the multi argument method of displaying multiple things and then the f-string